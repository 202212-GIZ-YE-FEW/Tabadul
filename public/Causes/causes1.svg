<svg width="115" height="113" viewBox="0 0 115 113" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M57.2339 7.62939e-06C25.6285 7.62939e-06 0 25.2999 0 56.5C0 87.7001 25.6285 113 57.2339 113C88.8392 113 114.468 87.7001 114.468 56.5C114.468 25.2999 88.8392 7.62939e-06 57.2339 7.62939e-06ZM85.8508 56.5C85.8303 63.9861 82.8087 71.1599 77.4464 76.4534C72.0842 81.7469 64.8172 84.7298 57.2339 84.75C49.6515 84.7266 42.3864 81.7427 37.0248 76.4499C31.6633 71.1571 28.6406 63.9852 28.6169 56.5C28.6439 49.0158 31.6675 41.8458 37.0284 36.5536C42.3893 31.2615 49.6525 28.2766 57.2339 28.25C64.8163 28.2734 72.0813 31.2573 77.4429 36.5501C82.8045 41.8429 85.8271 49.0148 85.8508 56.5ZM70.6675 27.0958C66.563 25.2595 62.0252 24.2143 57.2339 24.2143C52.4426 24.2143 47.9047 25.2595 43.8003 27.0958L31.7484 15.1985C39.4128 10.5427 48.2347 8.07565 57.2339 8.07144C66.5752 8.07144 75.2789 10.7068 82.7193 15.1985L70.6675 27.0958ZM27.4477 69.7614L15.3959 81.6586C10.8458 74.3177 8.17627 65.7216 8.17627 56.5C8.17627 47.2784 10.8458 38.6823 15.3959 31.3414L27.4477 43.2386C25.5917 47.2905 24.5288 51.7701 24.5288 56.5C24.5288 61.2299 25.5917 65.7095 27.4477 69.7614ZM43.8003 85.9042C47.9047 87.7405 52.4426 88.7857 57.2339 88.7857C62.0252 88.7857 66.563 87.7405 70.6675 85.9042L82.7193 97.8015C75.0549 102.457 66.2331 104.924 57.2339 104.929C47.8925 104.929 39.1888 102.293 31.7484 97.8015L43.8003 85.9042ZM87.02 43.2386L99.0718 31.3414C103.622 38.6823 106.291 47.2784 106.291 56.5C106.291 65.7216 103.622 74.3177 99.0718 81.6586L87.02 69.7614C88.876 65.7095 89.9389 61.2299 89.9389 56.5C89.9389 51.7701 88.876 47.2905 87.02 43.2386Z" fill="#F07167"/>
</svg>
