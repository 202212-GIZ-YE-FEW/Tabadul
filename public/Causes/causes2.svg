<svg width="125" height="113" viewBox="0 0 125 113" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M67.1865 6.96618C73.4164 2.52755 80.643 0.308238 88.8042 0.308238C93.2897 0.308238 98.2113 1.47954 103.569 3.94545C108.927 6.3497 113.163 9.18549 116.278 12.4528C122.695 20.3437 125.373 29.5292 124.439 40.0093C123.442 50.4894 119.954 58.6885 113.911 64.4834L66.6258 111.274C65.4421 112.445 63.947 113 62.2026 113C60.4582 113 59.0254 112.445 57.8417 111.274C56.658 110.103 56.0973 108.685 56.0973 106.959C56.0973 105.232 56.658 103.753 57.8417 102.582L86.4368 74.2853C87.9943 72.9291 87.9943 71.5112 86.4368 69.97C84.8793 68.4288 83.4465 68.4288 82.0759 69.97L53.4808 98.2662C52.2971 99.4375 50.8642 99.9924 49.0576 99.9924C47.3132 99.9924 45.8803 99.4375 44.6967 98.2662C43.513 97.0949 42.9523 95.677 42.9523 93.9509C42.9523 92.2248 43.513 90.7452 44.6967 89.5739L73.2918 61.2777C74.9738 59.7365 74.9738 58.1953 73.2918 56.6541C71.8589 55.1129 70.4883 55.1129 68.9309 56.6541L40.3358 85.2586C39.1521 86.4299 37.5946 86.9847 35.9126 86.9847C34.1682 86.9847 32.673 86.4299 31.3647 85.2586C30.1188 84.0873 29.4958 82.6694 29.4958 80.9433C29.4958 79.2171 30.1811 77.6759 31.5516 76.3197L60.209 47.9618C61.7665 46.4206 61.7665 45.0027 60.209 43.6465C58.6516 42.2902 57.1564 42.2902 55.7858 43.6465L26.8792 72.066C25.5087 73.299 24.0758 73.9771 22.5183 73.9771C20.774 73.9771 19.2788 73.3606 18.1574 72.066C16.9737 70.8331 16.3508 69.3535 16.3508 67.6274C16.3508 65.9013 16.9737 64.4834 18.1574 63.3121C37.2208 44.3246 48.9953 32.8582 53.4808 28.6045L75.6591 50.3044C78.0888 52.5854 80.8922 53.7567 84.3186 53.7567C88.6796 53.7567 92.106 52.0305 94.6602 48.5783C96.4046 46.0507 97.0276 43.2766 96.5292 40.1942C96.0308 37.1118 94.7225 34.5226 92.6044 32.365L67.1865 6.96618ZM80.02 45.9891L53.4808 19.6656L9.3733 63.3121C4.14021 58.072 1.14987 50.0578 0.464588 39.1462C-0.220698 28.2962 2.76964 18.8642 9.3733 10.9733C16.7868 3.69886 25.6333 0 35.9126 0C46.2541 0 55.0383 3.69886 62.2026 10.9733L88.8042 37.2968C89.9878 38.4681 90.5485 39.886 90.5485 41.6121C90.5485 43.3382 89.9878 44.8178 88.8042 45.9891C87.6205 47.0987 86.1876 47.7152 84.3186 47.7152C82.6366 47.7152 81.2037 47.0987 80.02 45.9891Z" fill="#F07167"/>
</svg>
