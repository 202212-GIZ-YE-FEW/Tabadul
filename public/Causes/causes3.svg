<svg width="129" height="114" viewBox="0 0 129 114" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M101.597 63.3786L77.7066 82.2277H45.6481V75.9025H71.7116C74.2699 75.9025 75.8023 73.0878 74.3789 70.9878L68.6853 62.5626C66.313 59.0395 62.3121 56.9269 58.0227 56.9269H45.6481H7.17804C3.63238 56.9269 0.766357 59.7542 0.766357 63.2521V101.203C0.766357 108.186 6.51123 113.854 13.5897 113.854H75.392C80.9381 113.854 86.2149 111.494 89.8695 107.377L129 63.2521L119.69 60.1907C113.503 58.154 106.694 59.3558 101.597 63.3786ZM86.7022 56.9269C97.3263 56.9269 105.937 48.4321 105.937 37.9512C105.937 20.8226 86.7022 0 86.7022 0C86.7022 0 67.4671 20.6898 67.4671 37.9512C67.4671 48.4321 76.0844 56.9269 86.7022 56.9269Z" fill="#F07167"/>
</svg>
