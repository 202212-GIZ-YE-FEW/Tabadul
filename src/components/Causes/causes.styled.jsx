import styled from "@emotion/styled";

export const CausesContainer = styled.div`
    width: 100%;
    background-color: #f1f1f1;
    display: flex;
    flex-direction: column;
    align-items: center;
    color: #3c4347;
`;
export const CausesWord = styled.h1`
    font-style: normal;
    font-weight: 700;
    font-size: 56px;
    line-height: 84px;
    top: 41px;
`;

export const ThreeCausesDiv = styled.div`
    width: 100%;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: row;
    flex-flow: wrap;
`;

export const SingleCause = styled.div`
    width: 400px;
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 10px;
`;
export const Title = styled.p`
    text-align: center;
    font-size: 32px;
    line-height: 48px;
    font-weight: 700;
`;
export const Paragraph = styled.p`
    text-align: center;
    font-size: 24px;
    line-height: 36px;
    font-weight: 400;
`;
