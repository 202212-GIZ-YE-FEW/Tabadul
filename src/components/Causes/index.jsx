import Image from "next/image";
import { useTranslation, withTranslation } from "next-i18next";
import React from "react";

import {
    CausesContainer,
    CausesWord,
    Paragraph,
    SingleCause,
    ThreeCausesDiv,
    Title,
} from "./causes.styled";

const Causes = () => {
    const { t } = useTranslation("causes");
    function translateMe(word) {
        return t(word);
    }
    return (
        <CausesContainer>
            <CausesWord> {translateMe("Causes")}</CausesWord>
            <ThreeCausesDiv>
                <SingleCause data-aos='fade-right'>
                    <Image
                        src='/Causes/causes1.svg'
                        width={100}
                        height={100}
                        alt='pic'
                    />
                    <Title>{t("sustainability")}</Title>
                    <Paragraph>{t("sustainabilityartical")}</Paragraph>
                </SingleCause>
                <SingleCause>
                    <Image
                        src='/Causes/causes2.svg'
                        width={100}
                        height={100}
                        alt='pic'
                    />
                    <Title>{t("cooperation")}</Title>
                    <Paragraph>{t("cooperationartical")}</Paragraph>
                </SingleCause>
                <SingleCause data-aos='fade-left'>
                    <Image
                        src='/Causes/causes3.svg'
                        width={100}
                        height={100}
                        alt='pic'
                    />
                    <Title>{t("giving")}</Title>
                    <Paragraph>{t("givingartical")}</Paragraph>
                </SingleCause>
            </ThreeCausesDiv>
        </CausesContainer>
    );
};

export default Causes;
