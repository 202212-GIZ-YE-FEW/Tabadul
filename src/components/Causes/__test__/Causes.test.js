import renderer from "react-test-renderer";

import Causes from "..";

it("renders correctly", () => {
    const tree = renderer.create(<Causes />).toJSON();
    expect(tree).toMatchSnapshot();
});
