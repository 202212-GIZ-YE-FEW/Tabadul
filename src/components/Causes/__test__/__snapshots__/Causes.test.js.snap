// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-1ityhfw"
>
  <h1
    className="css-1e6uix8"
  >
     
    Causes
  </h1>
  <div
    className="css-1xa8qvg"
  >
    <div
      className="css-ie2zpt"
      data-aos="fade-right"
    >
      <img
        alt="pic"
        data-nimg="1"
        decoding="async"
        height={100}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/Causes/causes1.svg"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={100}
      />
      <p
        className="css-99xlp1"
      >
        sustainability
      </p>
      <p
        className="css-7ho1qp"
      >
        sustainabilityartical
      </p>
    </div>
    <div
      className="css-ie2zpt"
    >
      <img
        alt="pic"
        data-nimg="1"
        decoding="async"
        height={100}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/Causes/causes2.svg"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={100}
      />
      <p
        className="css-99xlp1"
      >
        cooperation
      </p>
      <p
        className="css-7ho1qp"
      >
        cooperationartical
      </p>
    </div>
    <div
      className="css-ie2zpt"
      data-aos="fade-left"
    >
      <img
        alt="pic"
        data-nimg="1"
        decoding="async"
        height={100}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/Causes/causes3.svg"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={100}
      />
      <p
        className="css-99xlp1"
      >
        giving
      </p>
      <p
        className="css-7ho1qp"
      >
        givingartical
      </p>
    </div>
  </div>
</div>
`;
